tests/unit/store/traction/pacbio/plates/mutations.spec.js

```diff
@@ -1,29 +1,17 @@
 import mutations from '@/store/traction/pacbio/plates/mutations'
 import { dataToObjectById } from '@/api/JsonApi'
+import PacbioPlatesRequestFactory from '@tests/factories/PacbioPlatesRequestFactory'
 
-let plates
+const pacbioPlatesRequestFactory = PacbioPlatesRequestFactory()
 
 describe('mutations', () => {
-  beforeEach(() => {
-    plates = [
-      {
-        barcode: 'DN814327C',
-        created_at: '2021/06/03 06:59',
-        id: '61',
-      },
-      {
-        barcode: 'DN814567Q',
-        created_at: '2021/06/03 14:57',
-        id: '62',
-      },
-    ]
-  })
-
   it('"setPlates" sets "state.plates" to the given plates', () => {
     const state = {
       plates: {},
     }
-    mutations.setPlates(state, plates)
-    expect(state.plates).toEqual(dataToObjectById({ data: plates, includeRelationships: true }))
+    mutations.setPlates(state, pacbioPlatesRequestFactory.storeData)
```

Comment:
You can also move this function to the storeData function. I just added a selected which pulls out a small number of records.